src/DIRAC/WorkloadManagementSystem/Service/BasicPilotLoggingPlugin.py

```diff
@@ -0,0 +1,50 @@
+"""
```

Comment:
These plugins should not be in this directory, puy them in src/DIRAC/WorkloadManagementSystem/Client/PilotLoggingPlugins/